src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java

```diff
@@ -232,32 +234,56 @@ AstNode relations5( AstNode previous ){
 	
 	@Override
 	public AstNode additiveExpression( ){
-		// TODO Auto-generated method stub
-		return super.additiveExpression( );
+		if( Selector.SUMMAND.contains(LEXER.lookahead().TYPE) ){
```

Comment:
You don't need to check the selection set here because there is only one applicable rule and the call to `multiplicativeExpression` already checks for incorrect input. If you want to check it anyway, you have to use the selection set of `multiplicativeExpression`, not that of rule `summand`. If you remove the query, at least the end-to-end test will pass. The other test error is my fault 😉

Reply:
I have to correct myself: The test did not fail because of this. However, the statement is still correct.